sprint10 - js functii/tema3.js

```diff
@@ -0,0 +1,35 @@
+//Scrieți o funcție pentru a găsi numărul de apariții al unei litere într-un șir
+
+//Funcția letterCount() returnează numărul apariții al unei litere trimise ca parametru dintr-un șir dat.
+
+//Exemplu 1:
+
+//Input: sir = “Îmi place programarea” litera = “a”
+
+//Output: 4
+
+//Exemplu 2:
+
+//Input: sir = “Vreau să lucrez în IT”, litera = “r”
+
+//Output: 2  
+
+function letterCount(sir, litera) {
+    // Transformăm atât șirul cât și litera într-un format uniform 
+    litera = litera.toLowerCase();
+
+    let count = 0;
+
+    // Parcurgem șirul și numărăm aparițiile literei
+    for (let i = 0; i < sir.length; i++) {
+        if (sir[i] === litera) {
```

Comment:
```suggestion
        if (sir[i].toLowerCase() === litera) {
```
e nevoie sa verificam si fiecare litera sa fie mica.